docs/developers/deployment_guides/blockchain/oracles.md

```diff
@@ -43,6 +43,12 @@ You will also  need the following:
 
 - [SP solutions](https://github.com/Super-Protocol/solutions) - Super Protocol repository with solution examples, including the Oracle service.
 
+- Polygon Testnet wallets - you will need to create new wallets, **receive their addresses and private keys**, you can simply create new ones in MetaMask. 
```

Comment:
- Polygon Testnet wallets - we highly recommend that you to use two different wallets on Polygon testnet: 
    - For deploying "x509 verifier" smart contract, 
    - For deploying Oracle smart contract. 
  - But if you are going to use a single wallet, please make sure that you do not use it for any other operations, as they might disrupt the oracle. **Save their addresses and private keys**, you will need them later in the guide.